corehq/apps/registration/views.py

```diff
@@ -71,13 +71,16 @@ def register_user(request, domain_type=None):
                 activate_new_user(form, ip=get_ip(request))
                 new_user = authenticate(username=form.cleaned_data['email'],
                                         password=form.cleaned_data['password'])
-                login(request, new_user)
-                track_workflow(new_user.email, "Requested new account")
+
                 meta = {
                     'HTTP_X_FORWARDED_FOR': request.META.get('HTTP_X_FORWARDED_FOR'),
                     'REMOTE_ADDR': request.META.get('REMOTE_ADDR'),
-                }
+                }  # request.META can't be pickled for use by the task, so we copy the pertinent properties
                 track_created_hq_account_on_hubspot.delay(new_user, request.COOKIES, meta)
+                track_workflow(new_user.email, "Requested new account")
```

Comment:
There is a different task which is called on a signal hook on login(), which was always getting triggered before people were signing up. It was the task that `login()` triggers, and the `track_created_hq_account_on_hubspot` that was confusing people (since it looked like the logged in before they had created an account). 
It's not super critical that they always be in the right order, but I was getting emails about it, so thought I'd make a quick fix.